app/views/content/subjects/engineers-teach-physics/_header.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= render SubjectSpecific::HeaderComponent.new(
+  title: ["Teaching", "physics"],
```

Comment:
We don't mention engineering in the title - should we? (SEO / link to the pilot and comms etc.)